input/field_management/Barnyard_field_1.json

```
@@ -0,0 +1,14 @@
+{
+    "soil_specification": "example_soil.json",
+    "crop_specification": "example_rotation.json",
+    "fertilizer_management_specification": "example_fertilizer_schedule.json",
+    "manure_management_specification": "example_manure_schedule.json",
+    "tillage_management_specification": "example_tillage_schedule.json",
```

```suggestion
    "soil_specification": "soil_1",
    "crop_specification": "crop",
    "fertilizer_management_specification": "manure_schedule",
    "manure_management_specification": "fertilizer_schedule",
    "tillage_management_specification": "tillage_schedule",
```
These should be the name of the metadata blob that contains the corresponding specification. I had a conversation with Pooya before I merged the Crop and Soil metadata about this and he recommended using the blob names instead of file paths. This is reflected in the current example metadata on the `master` branch.